core/jobs_registry.py

```diff
@@ -52,11 +52,13 @@
     exp_jobs_one_off.ExplorationMigrationJobManager,
     exp_jobs_one_off.ExplorationValidityJobManager,
     exp_jobs_one_off.HintsAuditOneOffJob,
+    exp_jobs_one_off.InteractionAuditOneOffJob,
     exp_jobs_one_off.ItemSelectionInteractionOneOffJob,
     exp_jobs_one_off.ViewableExplorationsAuditJob,
     exp_jobs_one_off.ExplorationContentValidationJobForCKEditor,
     exp_jobs_one_off.InteractionCustomizationArgsValidationJob,
     exp_jobs_one_off.TranslatorToVoiceArtistOneOffJob,
+    exp_jobs_one_off.VoiceoverDurationSecondsOneOffJob,
```

Comment:
This job should be deleted in its current state, let's wait for @mzaman07 to add it to his PR and have it go through a thorough review. Don't want it to get run by accident.